bower.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "stencil-select",
+  "version": "0.1.0",
+  "description": "A styleable, accessible <select> component.",
+  "authors": [
+    "Mobify"
+  ],
+  "keywords": [
+    "mobify",
+    "components",
+    "patterns"
+  ],
+  "main": "",
+  "dependencies": {
+    "spline": "1.0.1"
+  },
+  "devDependencies": {
+    "vellum": "~1.0.0",
+    "modularized-normalize-scss": "~3.0.1"
+  },
```

Comment:
We need to sort our our dependency philosophy. Do we want every component to depend on normalize, vellum and spline, or is that too heavy-handed? What is the alternative?

In any case, vellum and normalize as development dependencies but not hard dependencies seems strange.